### ISSUE TITLE
Cannot install slate@0.24: slate-dev-logger has not been published to npm

### ISSUE THREAD

**Issue by ingro** [ianstormtaylor/slate#1115]
See the title, trying to `npm install slate` returns this error:

```bash
npm ERR! code ENOVERSIONS
npm ERR! No valid versions available for slate-dev-logger
```

**Comment by ianstormtaylor** [comment]
Thanks for reporting this @ingro. I'm not exactly sure what happen, since Lerna said it successfully published last night. But I've just run it again and it has now published `slate-dev-logger@0.1.1`. Thank you!

**Comment by ingro** [comment]
Thanks, it seems fixed now! :)
